pkg/controller/model/grafana_router.go

```diff
@@ -105,8 +106,8 @@ func createRouterContainer(cr *v1alpha1.Grafana) corev1.Container {
 			},
 		},
 		Resources:                resources,
-		LivenessProbe:            getRouterProbe(30, 20, cr.Namespace),
-		ReadinessProbe:           getRouterProbe(32, 10, cr.Namespace),
+		LivenessProbe:            getRouterProbe(30, 20, 10, cr.Namespace),
+		ReadinessProbe:           getRouterProbe(30, 20, 10, cr.Namespace),
```

Comment:
any idea why previous `delay` was `32`?

Reply:
I will make it to 32 for readiness probe. It was typo error.
Both readiness and liveness probe hit the same IAM api that's the reason they mentioned 2 delay so that  it will not hit at same time

Reply:
I have made changes and reverted to 32s for readiness probe